set.mm

```diff
@@ -80750,14 +80750,14 @@ currently used conventions for such cases (see ~ cbvmpox , ~ ovmpox and
       ( wrel wf1 wfo wa ccnv ctpos wf1o tposf12 tposfo2 anim12d df-f1o 3imtr4g
       ) ADZABCEZABCFZGAHZBCIZEZSBTFZGABCJSBTJPQUARUBABCKABCLMABCNSBTNO $.
 
-    $( The domain and range of a transposition.  (Contributed by NM,
+    $( The domain and codomain of a transposition.  (Contributed by NM,
```

Comment:
Non-blocking: Maybe here the codomain/range alternative is appropriate, too.  As in fofi.

Reply:
Done.

Reply:
Hmm, interesting. I wrote my above comments about onto functions before I saw "codomain/range" as an option, but I suppose "codomain/range" does have the virtue of emphasizing they are the same if the function is onto.